backend/webhook/consumer/pkg/worker/worker.go

```diff
@@ -0,0 +1,135 @@
+package worker
+
+import (
+	"errors"
+
+	beanstalk "github.com/beanstalkd/go-beanstalk"
+
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"log"
+	"net/http"
+	"time"
+)
+
+type ConsumerError struct {
+	Err error
+	T   time.Time
+}
+
+type Task struct {
+	done      chan bool
+	beanstalk *beanstalk.Conn
+	errors    []ConsumerError
+}
+
+type AiryMessage struct {
+	Endpoint string
+	Headers  map[string]string
+	Body     map[string]interface{}
+}
+
+var backoffSchedule = []time.Duration{
+	0 * time.Second,
+	1 * time.Second,
+	3 * time.Second,
+	10 * time.Second,
+}
```

Comment:
this works better as a function (I would probably use https://github.com/jpillora/backoff)